scripts/screens/relation_screens.py

```diff
@@ -2771,7 +2771,11 @@ def draw_info_block(self, cat, starting_pos: tuple):
             col1 += " moon"
         else:
             col1 += " moons"
-        col1 += "\n" + cat.trait
+        if len(cat.trait) > 11:
+            _t = cat.trait[:10] + ".."
+        else:
+            _t = cat.trait
+        col1 += "\n" + _t
         self.selected_cat_elements["col1" + tag] = pygame_gui.elements.UITextBox(col1,
                                                                                  scale(pygame.Rect((x + 42, y + 252),
                                                                                                    (160, -1))),
```

Comment:
Rather than shortening the trait, could we just make col1 larger? changing the x dimension to 180 seems to do the trick. 
![image](https://user-images.githubusercontent.com/115410010/228600485-009d6a9d-eb50-4c6f-afd4-b9a85ef1dd06.png)

Though it might be good to do both, since I believe "attention-seeker" takes more space than "compassionate"? or it might be about the same, it's good to have that redundancy regardless.  I'd just prefer for as much of the word to be shown as possible.

Reply:
This is a good point! I have updated the box size and increased the shortening cutoff.